### PR TITLE
Fix the Collection and related content presentation bug with a collec…

### PR DESCRIPTION
**Pull request by hoseinmirian**
to check that is not now showing collection data for a collection record please test:
http://localhost:8080/#/3474

to see if still working properly with other records test any random record or array express as a suggestion
http://localhost:8080/#/FAIRsharing.6k0kwd

### CODE REVIEW

**Review by knirirr — CHANGES_REQUESTED** [review]
It seems that nothing at all is turning up in "related content" for collections. is `prepareAssociations` still filtering out any "collects" or "recommends" relations from these data?

**Comment by knirirr** [comment]
Clarification from our meeting:

"in collections/recommendations" => Any reverseRecordAssociation where the label is "collects" or "recommends".
"related content" => Any recordAssociation where the label is NOT "deprecates".

File: src/components/Records/Record/RelatedContent.vue [hunk 1]
```diff
@@ -132,8 +132,8 @@ export default {
       tabsData: {
         selectedTab: 0,
         tabs: {
-          related_standards: {relation: ['collects', 'recommends'], registry: "Standard", data: []},
-          related_databases: {relation: ['collects', 'recommends'], registry: "Database", data: []}
+          related_standards: {relation: ['collects', 'recommends', 'deprecates'], registry: "Standard", data: []},
```
Comment (knirirr) [inline src/components/Records/Record/RelatedContent.vue]:
```suggestion
          related_standards: {relation: ['deprecates'], registry: "Standard", data: []},
```

File: src/components/Records/Record/RelatedContent.vue [hunk 2]
```diff
@@ -132,8 +132,8 @@ export default {
...
+          related_databases: {relation: ['collects', 'recommends', 'deprecates'], registry: "Database", data: []}
```
Comment (knirirr) [inline src/components/Records/Record/RelatedContent.vue]:
```suggestion
          related_databases: {relation: ['deprecates'], registry: "Database", data: []}
```

**Review by knirirr — CHANGES_REQUESTED** [review]
With the changes I've made a collection will show standards and databases that it collects properly.
But, collections can also collect policies and other collections; you might well have to add two more tabs to cover this. Or, simply a third tab for "other related records", to save space.

**Comment by knirirr** [comment]
Further comments at https://pyroeis.oerc.ox.ac.uk/#narrow/pm-with/10,18-pm/near/28168

**Comment by hoseinmirian** [comment]
New tab added for relatedContent section name "Other Related Records" that collects any other records excluding those with registries standards and databases. the tab is responsive for mobile and tablet and will pop the arrow button if the screen size is small. 

image


for testing a non-collection record to see there is no other related records tab please check:
http://localhost:8080/#/1827

and to see the tab shows up in a collection record please check:
http://localhost:8080/#/3449

**Review by knirirr — APPROVED** [review]

**Pull request merged**